toolkit/docs/building/prerequisites-mariner.md

```diff
@@ -0,0 +1,21 @@
+
+# Build Requirements
```

Comment:
Let's mention the OS in this header, so that it's clear what are these requirements for. Same for the Ubuntu version.

Reply:
Thanks for you advice.
Done!